src/vivarium/framework/lookup/interpolation.py

```diff
@@ -218,21 +228,21 @@ def check_data_complete(data, continuous_parameters) -> None:
         If a parameter contains non-continuous bins.
     """
 
+    sub_tables: list[tuple[tuple[Any, ...] | None, pd.DataFrame]]
     param_edges = [
-        p[1:] for p in continuous_parameters if isinstance(p, (Tuple, List))
+        p[1:] for p in continuous_parameters if isinstance(p, (tuple, list))
```

Comment:
Can't each item in `continuous_parameters` _only_ be a tuple or list anyway? (as per the definition of _ParameterType). So I don't think we even need the `if isinstance()` part?